channels.md

```diff
@@ -0,0 +1,68 @@
+
+Apart from that, other channels should be more appropriate. But don't let that make you lazy – check your email.
+
+## Slack
+
+Slack is our main communication channel. 
+
+Use Slack to contact anybody. Although it can be used for real-time communication, Slack is asynchronous, meaning people don't need to be available at the same time to participate in the conversation.
+
+Feel free to message people at any time. And they will answer when they have the time. If it’s after-hours for them expect a response on their next working day.
+
+Try to keep up with the channels you are in. If there has been a lot of activity while you were away, try to at least skim over it.
+
+Read all the mentions you get.
+
+Spend time getting to know Slack well. It's a tool we use a lot on a day-to-day basis and learning its keyboard shortcuts as well as properly configuring its notifications is key to a productive workflow.
+
+Reduce interruptions. Our chat is noisy and can be very distracting. It is strongly advisable you turn your Slack desktop notifications OFF and make a habit of checking your Slack when you are not focused on the tasks you should be working on. 
+
+If you don't want to do that, you can still pause notifications for a given period of time. Or alternatively schedule “Do not disturb” mode and Slack will let your coworkers know that you are not receiving notifications at the moment. What's more they’ll have the option to explicitly override this at their discretion.
+
+[Read the Slack guidelines.](slack.md)
+
+## Zoom
+
+We use Zoom for meetings. It’s a tool for video conferencing. You can start a meeting from Slack by typing `/zoom` in a channel or in a private conversation.
+
+Mute your microphone when you're not talking. It vastly improves quality for other listeners. Zoom allows you to have it muted by default so you don’t disturb the meeting when you join.
+
+Don't be late. If you can't avoid it, let other people know, so they’re not wondering where you are and when (and if) to start the meeting.
+
+When you're scheduling a meeting, consider the fact that this is the most expensive form of communication. The more people in the meeting, the less doing actual work, so have an agenda and a goal.
+
+## Google calendar
+
+We use Google calendar to organize our time.
+
+Please respect other people’s calendars when inviting them to meetings. Use the calendar’s option to ‘Find a time’ to do that. It’s available in the dialog, when creating a new event.
+
+Think about the length of your meetings. Leave 5-10 minutes between events. And build in break time if you plan a longer event, like a workshop or a retrospective.
+
+There is an option in Google Calendar’s Settings called ‘Speedy meetings’. It will set the default duration to 25 or 50 minutes. Use it to avoid back-to-back meetings.
+
+Allow guests to ‘Modify event’, so people can fix the zoom link or invite others without having to chase you.
```

Comment:
To add to the end:

> Make sure your attendance status is up-to-date. Set it to "not attending" if you're not gonna be on the call.

Reply:
I've updated all your other remarks with ninja commits. I'm not sure about the wording on this one. And it's getting a little bit duplicated with the line above. Let me think about another version